Makefile

```diff
@@ -33,6 +34,7 @@ stop-services:
 #-----------------------------------------------------------------------------
 build-test-fuseki:
 	@ echo -e '$(BUILD_PRINT)Building the Fuseki service'
+	@ docker volume create rdf-differ-template
```

Comment:
Fuseki does not need this volume

Reply:
fuseki doesn't, but `docker-compose` does, and it won't start without having all defined external volumes created in the `volumes` in the `docker-compose.yml`